R/redcapConnection.R

```diff
@@ -901,7 +901,7 @@ offlineConnection <- function(meta_data = NULL,
       has_user_roles = function() !is.null(this_user_roles), 
       flush_user_roles = function() this_user_roles <<- NULL, 
       refresh_user_roles = function(x) {this_user_roles <<- validateRedcapData(data = .offlineConnection_readFile(x), 
-                                                                               redcap_data = REDCAP_USER_ROLE_STRUCTURE)}, 
+                                                                               redcap_data = redcapUserRoleStructure("14.4.0"))}, 
```

Comment:
can use `this_version` to replace "14.4.0"

Reply:
done